### PR TITLE
Feature/security module

### PR DESCRIPTION
**Pull request by dariuszbolkowski**

### CODE REVIEW

**Comment by qwiatu-lp** [comment]
mv security-stack/security-stack-app ../
mv security-stack/security-stack-crd ../
rm -rf security-stack

File: charts/security-stack/security-stack-app/1.0.0/values.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  sealed-secrets:
+    enabled: true
+  external-secrets:
+    enabled: true
+  kyverno:
+    enabled: true 
+  kubearmor:
+    enabled: true
+  kubewarden-defaults:
+    enabled: true
```
Comment (qwiatu-lp) [inline charts/security-stack/security-stack-app/1.0.0/values.yaml:14]:
```suggestion
    enabled: false
```

File: charts/security-stack/security-stack-app/1.0.0/values.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  conjur-oss:
+    enabled: true
+  sealed-secrets:
+    enabled: true
+  external-secrets:
+    enabled: true
+  kyverno:
+    enabled: true 
+  kubearmor:
+    enabled: true
```
Comment (qwiatu-lp) [inline charts/security-stack/security-stack-app/1.0.0/values.yaml:12]:
```suggestion
    enabled: false
```

**Comment by qwiatu-lp** [comment]
suggestion: move cert-manager from security-stack-crd to security-stack-app

**Review by qwiatu-lp — COMMENTED** [review]
suggestion: kubearmor and kubewarden don't debploy by default

File: charts/security-stack/security-stack-app/1.0.0/questions.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    type: string
+    group: sealed-secrets
+    required: true
+- variable: global.kyverno.enabled
+  default: "true"
+  label: Deploy kyverno
+  type: boolean
+  group: "Components"
+- variable: global.kubearmor.enabled
+  default: "true"
```
Comment (qwiatu-lp) [inline charts/security-stack/security-stack-app/1.0.0/questions.yaml:35]:
```suggestion
  default: "false"
```

File: charts/security-stack/security-stack-app/1.0.0/questions.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  label: Deploy kubearmor
+  type: boolean
+  group: "Components"
+- variable: global.kubewarden-defaults.enabled
+  default: "true"
```
Comment (qwiatu-lp) [inline charts/security-stack/security-stack-app/1.0.0/questions.yaml:40]:
```suggestion
  default: "false"
```

File: charts/security-stack/security-stack-crd/1.0.0/values.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+
+global:
+  crd:
+    enabled: true 
+  kubearmor-operator:
+    enabled: true
```
Comment (qwiatu-lp) [inline charts/security-stack/security-stack-crd/1.0.0/values.yaml:6]:
```suggestion
    enabled: false
```

File: charts/security-stack/security-stack-crd/1.0.0/values.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  kubewarden-crds:
+    enabled: true
```
Comment (qwiatu-lp) [inline charts/security-stack/security-stack-crd/1.0.0/values.yaml:8]:
```suggestion
    enabled: false
```

File: charts/security-stack/security-stack-crd/1.0.0/values.yaml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  kubewarden-controller:
+    enabled: true
```
Comment (qwiatu-lp) [inline charts/security-stack/security-stack-crd/1.0.0/values.yaml:10]:
```suggestion
    enabled: false
```

**Review by qwiatu-lp — COMMENTED** [review]
feat: disable kubearmor and kubewarden

File: charts/security-stack/security-stack-crd/1.0.0/values.yaml [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  cert-manager-crd:
+    enabled: true
+  cert-manager:
+    enabled: true
```
[the remark this answers is not in the document]
Comment (dariuszbolkowski) [reply charts/security-stack/security-stack-crd/1.0.0/values.yaml:14]: Cannot be moved as some security-app components requires cert manager

**Pull request merged**